fw/ss_skb.h

```diff
@@ -77,6 +77,13 @@ struct tfw_skb_cb {
 
 #define TFW_SKB_CB(skb) ((struct tfw_skb_cb *)&((skb)->cb[0]))
 
+static inline bool
+ss_skb_is_within_fragment(char *begin_fragment, char *position,
+                          char *end_fragment)
+{
+	return begin_fragment <= position && end_fragment > position;
```

Comment:
position < end_fragment is better for me

Reply:
> position < end_fragment is better for me

I fixed it.